CLI "quarkus dev" command is not working with gradle kotlin

### Describe the bug

quarkus dev command is not working with gradle kotlin

### Expected behavior

should work

### Actual behavior

```
[ERROR] ❗  Unable to launch project in dev mode: Was not able to find a build file in: /Users/ia3andy/Downloads/code-with-quarkus
````

### How to Reproduce?

1. Generate some project with code.quarkus using Gradle Kotlin: https://stage.code.quarkus.io/?b=GRADLE_KOTLIN_DSL&e=resteasy&e=resteasy-jackson&e=hibernate-validator
2. Copy `quarkus dev` command in the popup
3. Unzip and cd
4.
```shell
$ quarkus dev
[ERROR] ❗  Unable to launch project in dev mode: Was not able to find a build file in: /Users/ia3andy/Downloads/code-with-quarkus
```

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/devtools/cli/src/main/java/io/quarkus/cli/build/GradleRunner.java
+++ b/devtools/cli/src/main/java/io/quarkus/cli/build/GradleRunner.java
@@ -235,9 +235,13 @@ public class GradleRunner implements BuildSystemRunner {
     }
 
     void verifyBuildFile() {
-        File buildFile = projectRoot.resolve("build.gradle").toFile();
-        if (!buildFile.isFile()) {
-            throw new IllegalStateException("Was not able to find a build file in: " + projectRoot);
+        for (String buildFileName : buildTool.getBuildFiles()) {
+            File buildFile = projectRoot.resolve(buildFileName).toFile();
+            if (buildFile.exists()) {
+                return;
+            }
         }
+        throw new IllegalStateException("Was not able to find a build file in: " + projectRoot
+                + " based on the following list: " + String.join(",", buildTool.getBuildFiles()));
     }
 }

